setup.py

```diff
@@ -43,6 +43,8 @@
     zip_safe=False,
     platforms='any',
     install_requires=[
+        "Flask-Admin>=1.0.9",
```

Comment:
so, I have to remove flask-admin and workflow from requirements.txt and also from setup.py?